content/cloud/changelog.md

```diff
@@ -0,0 +1,7 @@
+title = "Fermyon Cloud Changelog"
+template = "changelog"
+date = "2022-01-01T00:00:01Z"
+[extra]
+url = "https://github.com/fermyon/developer/blob/main//content/cloud/spin-vs-code-extension.md"
```

Comment:
This needs to be removed. apologies for letting it slip through.

Reply:
The whole file, or just `extra.url` part?

Reply:
Apologies for not being clear the first time. The `extra.url`. The file needs to exist for it to be rendered.